File: BE/gumid101/src/main/java/com/ssafy/gumid101/res/OtherUserFileDto.java
package com.ssafy.gumid101.res;

import java.util.List;

import com.ssafy.gumid101.dto.CrewTotalRecordDto;
import com.ssafy.gumid101.dto.ImageFileDto;
import com.ssafy.gumid101.dto.UserDto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.Setter;

@AllArgsConstructor
@Builder
@Setter
@Data
public class OtherUserFileDto {

	private UserDto user;
	private ImageFileDto imgFile;
	private CrewTotalRecordDto totalRecord;
	private List<AchieveCompleteDto> achieveList;
	
	
}

File: AOS/RunWithMe/app/src/main/java/com/ssafy/runwithme/model/response/OtherUserFileDto.kt
package com.ssafy.runwithme.model.response

import com.google.gson.annotations.SerializedName
import com.ssafy.runwithme.model.dto.ImageFileDto
import com.ssafy.runwithme.model.dto.UserDto

data class OtherUserFileDto(
    @SerializedName("user") var userDto : UserDto,
    @SerializedName("imgFile") var imgFileDto: ImageFileDto,
    @SerializedName("totalRecord") var totalRecord: CrewMyTotalRecordDataResponse,
    @SerializedName("achieveList") var achieveList: List<MyAchieveResponse>
)